Week-7/Problem 7.4 Check Parenthesis.js

```diff
@@ -0,0 +1,86 @@
+console.log("Problem 7.4: Check Parenthesis");
+console.log("Time complexity O(n) and space complexity O(n)");
+class Stack
```

Comment:
this can go in as seperate file

Reply:
Crated a sperate file for classes. 
classes.js